### PR TITLE
Change pin email format and added gettext

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
The Email in german is as such

Screenshot 2024-11-08 at 10 15 05

The Email in English is as such 

Screenshot 2024-11-08 at 10 16 05

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,3 +1,3 @@
-erlang 26.2.5
+erlang 24.1.1
```
Comment (wintermeyer) [inline .tool-versions]: Why do we downgrade Erlang here?

**Review by wintermeyer — CHANGES_REQUESTED** [review]
Please remove those duplicate empty lines. Just one empty line is plenty. And indent the PIN by 4 spaces.

Bildschirmfoto 2024-11-08 um 11 18 10

File: .tool-versions (hunk 1 above)
Comment (MICHAELMUNAVU83) [reply .tool-versions]: Aah my bad , let me remove this

**Comment by MICHAELMUNAVU83** [comment]
I now have it as such
Screenshot 2024-11-11 at 08 30 41

I have tried the pin indentation and it does not seem to work , we can have it as a seperate issue .

**Review by wintermeyer — APPROVED** [review]

**Comment by wintermeyer** [comment]
@MICHAELMUNAVU83 I just triggered a deployment. Please test it in 10-15 minutes on production.

**Comment by MICHAELMUNAVU83** [comment]
It is now as such
Screenshot 2024-11-11 at 14 16 21

**Pull request merged**